tests/unit/test_tools_changelog.py

```diff
@@ -55,9 +55,14 @@ def test_normalize_title(self):
             'Fixes #101 — Everything you can imagine is real.')
         expected = 'Fixes #101 - Everything you can imagine is real.'
         self.assertEqual(actual, expected)
-        # GreenKeeper title style
-        actual = changelog.normalize_title('Greenkeeper style 🚀')
-        expected = 'NPM update - Greenkeeper style.'
+        # # GreenKeeper title style
+        # actual = changelog.normalize_title('Greenkeeper style 🚀')
+        # expected = 'NPM update - Greenkeeper style.'
+        # self.assertEqual(actual, expected)
+
+        # Dependabot-review title style
+        actual = changelog.normalize_title('Bump grunt from 1.0.4 to 1.1.0')
+        expected = 'NPM update - grunt from 1.0.4 to 1.1.0.'
```

Comment:
This title reads a bit odd after removing the word `Bump`, maybe expected could be: `NPM update - bump grunt from 1.0.4 to 1.1.0`?

Reply:
👍 agreed with @miketaylr 
Though honestly the bump is not necessary very telling too. 

I would favor a `'NPM update - upgrade grunt from 1.0.4 to 1.1.0.'`
on the other hand, the proposal of mike is requiring less work. :)

Reply:
> +1 agreed with @miketaylr
> Though honestly the bump is not necessary very telling too.
> 
> I would favor a `'NPM update - upgrade grunt from 1.0.4 to 1.1.0.'`
> on the other hand, the proposal of mike is requiring less work. :)

Yoss! I guess upgrade is a better term to use instead of bump.

Reply:
Or should we follow the old trend
"Update grunt to latest version"?

Reply:
having the version numbers can help figure out the regressions.